examples/nlp/sentence_embeddings_with_sbert.py

```diff
@@ -75,12 +82,63 @@
 
 For our example, the subnetwork will be a RoBERTa model that has a pooling layer on top
 of it to produce the embeddings of the input sentences. These embeddings will then be
-compared to each other to learn to produce semantically meaningful embeddings.
+compared with each other to learn to produce semantically meaningful embeddings.
 
 The pooling strategies used are mean, max, and CLS pooling. Mean pooling produces the
 best results. We will use it in our examples.
+
+We will use two methods for fine-tuning:
+
+- Fine-tune using the regression objective function
+- Fine-tune using the triplet Objective function
 """
 
+"""
+### Build the encoder model.
```

Comment:
I think this section made more sense where it was, in logical order:
- Convert text to tokens.
- Encode the tokens.